//@ known-failure
fn foo() {
    let _v = vec![1];
}
